R/repository.R

```diff
@@ -79,17 +79,32 @@
 as.data.frame.git_repository <- function(x, ...) {
     do.call("rbind", lapply(commits(x), as.data.frame))
 }
+##' @export
+as.data.frame.data_repository <- function(x, ...) {
+    NextMethod()
+}
 
 ##' Open a repository
 ##'
 ##' @param path A path to an existing local git repository.
 ##' @param discover Discover repository from path. Default is TRUE.
-##' @return A \code{git_repository} object with entries:
+##' @param project The name of of project. Refers to a local path in case of a
+##' data repository. Defaults to \code{NULL}, indicating a standard repository.
+##' @return Either a \code{git_repository} object with entries:
 ##' \describe{
 ##'   \item{path}{
 ##'     Path to a git repository
 ##'   }
 ##' }
+##' or a \code{data_repository} object with entries:
```

Comment:
I'm having mixed feelings about the double usage of the naming 'repository'... A repository is a `git` repository and this functions opens a git repository. As every function should do one thing, I would not use the same function to open a `data-repository` (which is part of a git-repository?!).